qiita_pet/uimodules/prep_template_tab.py

```diff
@@ -39,9 +39,12 @@ def _get_accessible_raw_data(user):
     raw_data
     """
     d = {}
-    for sid in user.user_studies:
+    accessible_studies = user.user_studies.union(user.shared_studies)
+    for sid in accessible_studies:
         for rdid in Study(sid).raw_data():
-            d[int(rdid)] = Study(RawData(rdid).studies[-1]).title
+            rdid = int(rdid)
+            if rdid not in d:
+                d[rdid] = Study(RawData(rdid).studies[-1]).title
```

Comment:
Not sure but note that you are getting all raw datas of which a Study has access and then adding the title of where it was used lastly. Just to be clear, not sure what that gives us.